doc/reference/kuadrant.md

```diff
@@ -16,6 +16,80 @@ The Kuadrant operator creates a Limitador CR named `limitador` in the same names
+| ports                  | [Ports](#ports) |      No      | Port numbers of the authorization server (gRPC and raw HTTP interfaces).                                        |
+| tls                    | [Tls](#tls)     |      No      | TLS configuration of the authorization server (gRPC and HTTP interfaces).                                       |
+| timeout                | Integer         |      No      | Timeout of external authorization request (in milliseconds), controlled internally by the authorization server. |
+| maxHttpRequestBodySize | Integer         |      No      | Maximum payload (request body) size for the auth service (HTTP interface0, in bytes.                            |
+
+##### Ports
+
+| **Field** | **Type** | **Required** | **Description**                                                                                        |
+|-----------|----------|:------------:|--------------------------------------------------------------------------------------------------------|
+| grpc      | Integer  |      No      | Port number of the gRPC interface of the authorization server. Set to 0 to disable this interface.     |
+| http      | Integer  |      No      | Port number of the raw HTTP interface of the authorization server. Set to 0 to disable this interface. |
+
+#### Metrics
+
+| **Field** | **Type** | **Required** | **Description**                                                                              |
+|-----------|----------|:------------:|----------------------------------------------------------------------------------------------|
+| deep      | Boolean  |      No      | Enable/disable metrics at the level of each evaluator config exported by the metrics server. |
+| port      | Integer  |      No      | Port number of the metrics server.                                                           |
+
+#### OIDCServer
+
+| **Field**  | **Type**    | **Required** | **Description**                                                               |
+|------------|-------------|:------------:|-------------------------------------------------------------------------------|
+| port       | Integer     |      No      | Port number of OIDC Discovery server for Festival Wristband tokens.           |
+| tls        | [TLS](#tls) |     Yes      | TLS configuration of the ODIC Discovery server for Festival Wristband tokens. |
+
+#### Tracing
+
+| **Field** | **Type** | **Required** | **Description**                                                                                     |
+|-----------|----------|:------------:|-----------------------------------------------------------------------------------------------------|
+| endpoint  | String   |     Yes      | Full endpoint of the OpenTelemetry tracing collector service (e.g. http://jaegar:14268/api/traces). |
+| tags      | Map      |      No      | Key-value map of fixed tags to add to all OpenTelemetry traces emitted by Authorino.                |
```

Comment:
Ref. https://github.com/Kuadrant/kuadrant-operator/pull/393#discussion_r1591051701

The latest version of the `Authorino` API shall include another field here, `insecure`.